book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :author, :title
+  attr_reader :rentals
+
```

Comment:
- [ ] Please add Setters and Getters `attr_accessor` for @rentals, not Getters only `attr_reader` as you did so the book class can add new rentals.